package.json

```diff
@@ -181,7 +181,7 @@
     "@fastify/fast-json-stringify-compiler": "^3.0.0",
     "abstract-logging": "^2.0.1",
     "avvio": "^8.1.3",
-    "fast-json-stringify": "^4.1.0",
+    "fast-json-stringify": "^4.2.0",
```

Comment:
I understand we can drop this?

Reply:
Yes, it needs to move back as `devDenpendencies`. Forgotten to type `-D` when install it again.